frontend/src/app/shared/pipes/values.pipe.ts

```diff
@@ -10,9 +10,11 @@ export class ValuesPipe implements PipeTransform {
      * @returns an array of the values in the object in their original order
      */
     // Accepts undefined and null too, to make it easier to use in templates with e.g. | async
-    transform<Value extends { [key: string]: Item } | null | undefined, Item>(
-        object: Value
-    ): Value extends null | undefined ? Value : Item[] {
+    transform<
+        Value extends { [key: string]: Item } | null | undefined,
+        // When the `:` side is used, `Item` is unused
```

Comment:
What is bad about that? I don't get any errors or warnings from TS or eslint about that...

Reply:
This is not bad. It's just a hint at the programmer, why the `Item` is `never` in this case. I think this would be a bit unintuitive for me.

Reply:
So you changed

```ts
    transform<Value extends { [key: string]: Item } | null | undefined, Item>(
        object: Value
    ): Value extends null | undefined ? Value : Item[] {
```

to

```ts
    transform<
        Value extends { [key: string]: Item } | null | undefined,
        // When the `:` side is used, `Item` is unused
        Item = Value extends { [key: string]: infer T } ? T : never
    >(object: Value): Value extends null | undefined ? Value : Item[] {
```

to make the code more understandable?

Reply:
The previous typings were not correct always correct